stable/insights-agent/templates/kyverno/cronjob.yaml

```diff
@@ -0,0 +1,36 @@
+{{- if and .Values.kyverno .Values.kyverno.enabled -}}
+{{- $_ := set . "Label" "kyverno" }}
+{{- $_ := set . "Config" .Values.kyverno }}
+{{- include "cronjob" . }}
+spec:
+  {{ include "cronjob-spec" . | nindent 2 | trim }}
+  jobTemplate:
+    spec:
+      {{ include "job-spec" . | nindent 6 | trim }}
+      template:
+        {{ include "job-spec-metadata" . | nindent 8 | trim }}
+        spec:
+          {{ include "job-template-spec" . | indent 10 | trim }}
+          containers:
+          - {{ include "container-spec" . | indent 12 | trim }}
+            env:
+            - name: FAIRWINDS_TOKEN
+              valueFrom:
+                secretKeyRef:
+                  {{ if .Values.insights.tokenSecretName -}}
+                  name: {{ .Values.insights.tokenSecretName }}
+                  {{ else -}}
+                  name: {{ include "insights-agent.fullname" . }}-token
+                  {{ end -}}
+                  key: token
+            - name: FAIRWINDS_INSIGHTS_HOST
+              value: {{ .Values.insights.host }}
+            - name: FAIRWINDS_ORG
+              value: {{ .Values.insights.organization | quote  }}
+            - name: FAIRWINDS_CLUSTER
+              value: {{ .Values.insights.cluster | quote }}
```

Comment:
🤔 do we need these?

Reply:
I was under the impression that the uploader container needs the host/cluster/org/token or am I reading that wrong? https://github.com/FairwindsOps/charts/blob/effe98e2992a0e5a6482e6e5600087b3ef4adbd2/stable/insights-agent/templates/_uploader.yaml

Reply:
Or am I adding duplication from the template here? https://github.com/FairwindsOps/charts/blob/effe98e2992a0e5a6482e6e5600087b3ef4adbd2/stable/insights-agent/templates/_uploader.yaml

Was odd that the OPA cronjob is set up like that

Reply:
ahh OPA needs some of these things because it _downloads_ policies via the API. This setup should be simpler